app/views/communicart_mailer/proposal_reminder.text.erb

```diff
@@ -0,0 +1,4 @@
+All unapproved FY15 credit card funding requests are being cancelled.
```

Comment:
given the content of this email, might it make sense to name it something like `proposal_cancellation_notification` instead?